### PR TITLE
Add 38c3 page

### PR DESCRIPTION
**Pull request by link2xt**

### CODE REVIEW

**Comment by link2xt** [comment]
Preview is here: https://staging.delta.chat/1010/38c3.html
Just testing the styles from https://events.ccc.de/congress/2024/infos/styleguide.html, otherwise page as a copy of 37c3

File: 38c3.html [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+      .qrcode {
+        max-width: 300px;
+        width: 60vw;
+      }
+    </style>
+  </head>
+
+  <body>
+    <h2 id="delta-chat-37c3">Delta Chat @ 37c3</h2>
```
Comment (hpk42) [inline 38c3.html:90]:
```suggestion
    <h2 id="delta-chat-38c3">Delta Chat @ 38c3</h2>
```

**Review by hpk42 — COMMENTED** [review]
i fixed one but there are several 37c3 references.  intentional?

**Comment by link2xt** [comment]
> Just testing the styles from https://events.ccc.de/congress/2024/infos/styleguide.html, otherwise page as a copy of 37c3

There is no content, I just copied 37c3 page and changed the styles.

**Comment by link2xt** [comment]
Let's merge it, it is not visible anyway.

**Pull request merged**